src/extra.ml

```diff
@@ -130,6 +130,27 @@ module Array =
       Array.length a1 = Array.length a2 && for_all2 eq a1 a2
   end
 
+module Ord =
```

Comment:
Why using a module here?

Reply:
I created the module for that since most of the code in extra.ml is organized in this fashion. It just seems a bit weird to include a separate function in a file written in this style.

Reply:
Right but, it seems to me, not for the same reasons. String, List and Arrays are extensions of already existing modules. Option is not but, as it provides functions with common names, it is not intented to be opened but used to qualify those common names.